### DISCUSSION TITLE
Recovery options for deleted docker-compose

### DISCUSSION THREAD

**Discussion by dahifi** [firefly-iii/firefly-iii#9182] [Firefly III]
Hi, 

So I come back from summer break and find that I've deleted the project folder where I had my docker-compose and other customized files for the data importer. 

My docker setup works, I can still log into Firefly and see my txs, but data importer relies on missing folders for uploads, and I can't upgrade. I think I'll need to recover the db passwords somehow, but I wanted to ask before I sit down and start from the installation instructions. 

Thanks,

**Comment by dahifi** [comment]
To answer my own question: I was able to retrieve the env vars from the existing docker containers, and managed to migrate my data to another machine by dumping and loading the db docker volume. 

Still having problems with the data importer, but I'll take those problems elsewhere.

**Reply by JC5** [reply]
Nice work, glad to hear you got your data back.
